arch/x86_64/fentry.S

```diff
@@ -1,8 +1,55 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = 8(%rsp) */
+/* stack frame (with -pg -mfentry): parent addr = 56(%rsp), child addr = 48(%rsp) */
```

Comment:
I meant you need to subtrace 48 from the offsets since it was adjusted by our `__fentry__`.